cocotb/share/lib/gpi_log/gpi_logging.c

```diff
@@ -159,14 +159,18 @@ void gpi_log(const char *name, long level, const char *pathname, const char *fun
         PyGILState_Release(gstate);
         return;
     }
-
-    if (filter_ret == Py_True) {
```

Comment:
Changing to `!=` would also have worked here, but this formulation makes it work if `isEnabledFor` returns `1` instead of `True.